livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -64,6 +64,7 @@ public void open() {
 
   @Override
   public void close() {
+    livyHelper.closeSession(userSessionMap, "spark");
```

Comment:
Have fixed this, https://github.com/apache/incubator-zeppelin/pull/827/files#diff-f79d9062c41ca03d243c15e3a60246fcR65. Strange this did not get refreshed.